mlir/lib/Dialect/Rock/Generator/ConvGenerator.cpp

```diff
@@ -215,37 +215,37 @@ LogicalResult ConvGenerator::hasValidDimension() const {
   }
 
   int64_t expectedOutHeight = outputDim(
-      inDim["h"], filDim["y"], config.paddingLeftDims[DIM::HEIGHT],
+      inDim["0"], filDim["0"], config.paddingLeftDims[DIM::HEIGHT],
```

Comment:
inDim and filDim are StringMaps;  the key strings change.  DIM::HEIGHT is part of an enum defined in ConvGenerator.h, because I didn't want to hard-code 0 and 1 everywhere.

Reply:
Commit 4d3dff009 does that, along with some places where, eg, llvm::any_of() is more concise than std::any_of().